test/directus_api_test.dart

```diff
@@ -142,7 +142,7 @@ void main() {
               operator: FilterOperator.between,
               value: [r"$NOW", r"$NOW(+2 weeks)"]));
       expect(request.url.toString(),
-          r'http://api.com/items/article?fields=*&filter=%7B+%22date%22%3A+%7B+%22_between%22%3A+%5B%22%24NOW%22%2C+%22%24NOW%28%2B2+weeks%29%22%5D+%7D%7D');
+          r'http://api.com/items/article?fields=*&filter=%7B+%22date%22%3A+%7B+%22_between%22%3A+%5B%22%24NOW%22%2C%22%24NOW%28%2B2+weeks%29%22%5D+%7D%7D');
```

Comment:
What is this change, and how is this related to the current PR ?

Reply:
The unit test wasn't passing since bdc0b7e14284e2686da28ed720ed539c14e71143. It has no relation with the main goal of this PR, but I prefer when the tests pass. Do you wish an other PR for that ?